Creating telnyx chatbot

RJSkorski opened the pull request:
This PR contains a chatbot example that shows how to integrate with Telnyx voice services. 

As part of it, the following changes were made:

- a new example with the readme of how to use it was created
- A new serializer that handles Telnyx bidirectional media streaming was added
- 2 methods for de/encoding PCMA were added to utils.py

aconchillo commented:
@RJSkorski This looks wonderful! Do you mind fixing the Ruff formatting before I merge? :pray:

RJSkorski commented:
@aconchillo, thank you for the review and the approval 🙇.  I adjusted the formatting and made one improvement: I moved the encoding setting from the serializer to the server script.

aconchillo commented:
Merged. Thank you @RJSkorski !
